tools/scoring/score/dimensions/field_selection.py

```diff
@@ -0,0 +1,36 @@
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the License);
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an AS IS BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+""" Core component """
+
+from typing import Any, Dict, List, Tuple
+
+from score.dimensions.dimension import Dimension
+
+
+class FieldSelection(Dimension):
+  """ Quantifies whether the correct fields were mapped (versus ignored) """
```

Comment:
can you please add an example in the docstring what is a field?
is it the raw field? or the standard field from the ontology ?
What is a selection? it is present in the proposed?

Reply:
Yes, successful "field selection" means "the raw fields were present in the proposed file."